model/Export/AbstractQTIItemExporter.php

```diff
@@ -316,4 +326,26 @@ protected function getMetadataExporter()
         }
         return $this->metadataExporter;
     }
+
+    private function copySharedStimulusCssFiles(string $link, string $basePath): string
+    {
+        $asset = new core_kernel_classes_Resource(UriHelper::decode($link));
```

Comment:
i would break this into a different method and i would eliminate the static call if possible

Reply:
We should use TraitI would use 
`getResource` from `\oat\generis\model\OntologyAwareTrait`